Prose/ProseLib/Sources/UnreadFeature/UnreadScreen.swift

```diff
@@ -32,6 +32,7 @@ public struct UnreadScreen: View {
             .background(Colors.Background.message.color)
             .toolbar(content: Toolbar.init)
             .onAppear { actions.send(.onAppear) }
+            .groupBoxStyle(.spotlight)
```

Comment:
This is just nitpicking, but as I said about the GroupBoxStyle belonging in the spotlight section, I'd find it logical that we apply the style there.

Reply:
Maybe the style should be renamed then. I don't think that the `MainScreen` should decorate some views and not others. It's better that `MainScreen` doesn't make any assumptions about what the content looks like and just displays it, no?

Reply:
I don't agree, you wrote:

```swift
List(selection: viewStore.binding(
    get: \.selection,
    send: SidebarAction.selection
)) {
    self.spotlightSection
    self.contactsSection
    self.groupsSection
}
.listStyle(.sidebar)
```

Why not

```diff
List(selection: viewStore.binding(
    get: \.selection,
    send: SidebarAction.selection
)) {
    self.spotlightSection
+         .groupBoxStyle(.spotlight)
    self.contactsSection
    self.groupsSection
}
.listStyle(.sidebar)
```

(Or, better, `.groupBoxStyle(.spotlight)` in `self.spotlightSection`, that was for the code snippet)

(And, I don't know why you mention `MainScreen`)

Reply:
[It's applied](https://github.com/prose-im/prose-app-macos/blob/988426165be8d566e68014ff41458942ea1fd54a/Prose/ProseLib/Sources/SidebarFeature/NavigationDestinationView.swift#L44) to the `UnreadScreen` at the moment. Are you sure that would be equivalent?

Reply:
I'll leave it in ProseUI everything else doesn't make sense. But at some point it should probably be renamed to _what it is_ not _where it originates from_.